src/ui/include/EditorNS/editor.h

```diff
@@ -265,17 +243,18 @@ namespace EditorNS
         QList<Theme> themes();
         void setTheme(Theme theme);
         static Editor::Theme themeFromName(QString name);
-
-        QList<Selection> selections();
-
+ 
         /**
          * @brief Returns the currently selected texts.
          * @return
          */
-        Q_INVOKABLE QStringList selectedTexts();
+        Q_INVOKABLE void setSelectionsText(const QStringList &texts, selectMode mode);
```

Comment:
Yea... that's what I meant... just going to get this ready for merging at this point... no more changes unless requested, lol.